salsa_dancing_molecules/simulations/simulation.py

```diff
@@ -1,4 +1,68 @@
 """Module for running a simulation."""
+from ..lennardjonesparse import parse_lj_params
+from ase.md.velocitydistribution import MaxwellBoltzmannDistribution
+from ase.md.verlet import VelocityVerlet
+from ase.md.andersen import Andersen
+from ase import units
+from asap3 import Trajectory
+from ..variables import Variables
+
+
+def choose_potential(potential, kim_model, use_asap, atoms):
+    """Change the atoms object according to what potential is used.
+
+    Args:
+        potential - String with name of potential to be used.
+        kim_model - String with ID for ensemble to be chosen from openkim.
+        use_asap - boolean if asap3 is to be used instead of ASE
```

Comment:
I'd prefer the documentation on the form `use_asap: bool - True if asap3 is to be used instead of ASE`. We have that formatting in a few other files.